lib/wfsc/falco_est.m

```diff
@@ -51,12 +51,24 @@
             ev = falco_est_borde_traub_probing(mp, ev);
             
         case{'scc'}
+            if ~mp.flagSim && strcmp(mp.testbed,'iact')
```

Comment:
1) I recommend using `strcmpi` instead of `strcmp`.
2) `mp.testbed` doesn't currently have a default, so this will throw an error if you run a simulation using the SCC. So I'd recommend putting a dummy value in falco_set_optional_variables.m like this `if(isfield(mp,'testbed')==false); mp.testbed = 'none'; end `